MEMENTO/Memento_Face_Detect_Recognize/memento_platformio_camera/platformio.ini

```diff
@@ -0,0 +1,31 @@
+; PlatformIO Project Configuration File for MEMENTO/PyCamera
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[env:adafruit_camera_esp32s3]
+platform = espressif32 @ ^6.5.0
+board = adafruit_camera_esp32s3
+framework = arduino
+upload_port = /dev/cu.usbmodem13301
+monitor_port = /dev/cu.usbmodem13301
+monitor_speed = 115200
+board_upload.before_reset = default_reset
+build_flags = -DBOARD_HAS_PSRAM -mfix-esp32-psram-cache-issue -DARDUINO_LOOP_STACK_SIZE=8092
+board_build.partitions = partitions.csv
+lib_deps = adafruit/Adafruit AW9523
+           adafruit/Adafruit NeoPixel
+           Wire
+           SPI
+           adafruit/Adafruit BusIO
+           adafruit/Adafruit GFX Library
+           adafruit/Adafruit ST7735 and ST7789 Library
+           adafruit/SdFat - Adafruit Fork
+           adafruit/Adafruit GFX Library
+           https://github.com/adafruit/Adafruit-ST7735-Library

```

Comment:
Do these two not work as non-git references? To be fair people doing dev need git...

Reply:
They do